Sources/Helpers/OAProfileSettingsResetHelper.h

```diff
@@ -0,0 +1,18 @@
+//
+//  OAProfileSettingsResetHelper.h
+//  OsmAnd
+//
+//  Created by nnngrach on 08.09.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OASettingsHelper.h"
+
+@interface OAProfileSettingsResetHelper : NSObject
```

Comment:
Not clear why do we need such helper class. @PaulStets please review it and maybe compare with android code.

Reply:
This class is indeed redundant, all the reset logic in Android is in ConfigureProfileFragment.java: 171 ```public void resetAppModePrefs(ApplicationMode appMode)```. The only difference is: unlike in Android all  the Rendering and Routing settings should be reset separately (see OASettingsHelper.mm import logic).